data/hai/hai reveal 3 early.txt

```diff
@@ -0,0 +1,470 @@
+			distance 4208.63
+			period 2158.5
+			object Mountaintop
+				sprite "planet/station hai eight geocoris dim"
+				distance 315
+				period 31.1964
+	planet Mountaintop
+		government "Hai"
+		attributes uninhabited
+		landscape land/station0
+		description `This station is made of eight Geocoris ships dismantled and reassembled into a spherical structure with external storage and landing platforms. Hai workers are shuffling about the station, installing human amenities and vast holographic projection systems. Presently it looks much like a construction site, but it is rapidly approaching completion.`
+		security 0.2
+
+
+
+mission "Hai Building Mountaintop [2] Recruits"
+	name "Encourage recruits for Mountaintop"
+	description "Now that the station is going ahead, the Hai need to recruit people to play the part. Someone needs to sell it to them, and apparently that someone is you."
+	landing
+	source "Hai-home"
+	stopover "Makerplace"
+	stopover "Darkmetal"
+	stopover "Giverstone"
+	stopover "Stonebreak"
+	stopover "Cloudfire"
+	stopover "Snowfeather"
+	stopover "Icelake"
+	stopover "Newhome"
+	stopover "Frostmark"
+	stopover "Heartvalley"
+	stopover "Allhome"
+	stopover "Dustmaker"
+	stopover "Greenbloom"
+	stopover "Skyfarm"
+	stopover "Farwater"
+	stopover "Greenwater"
+	destination "Mountaintop"
+	clearance
+	passengers 1
+	blocked `Terry contacts you and says, "Captain, you're going to need a bunk free for me. Do you have space?" You do not, so you should complete or abort another mission to make room for her, then return here.`
+	on visit
+		dialog `You have arrived at the Mountaintop construction site, but it isn't finished or staffed yet.`
+	to offer
+		has "Hai Building Mountaintop [1b] Return: done"
+	on offer
+		conversation
+			`At Hai-home, you are directed to a now familiar government hangar. Sayari, Xilin, and Alondo all send you greeting pings, but are busy at the time of your landing. To your surprise though, Terry is banging on your airlock before too long.`
+			`	"Captain <last>, or should I call you <first>? Anyway, doesn't matter. How would you like to make some money?" The smile she gives you is something like you imagine a large cat giving the tasty morsel it just identified as its food. "Of course you do, everyone likes to make money," she continues, not waiting for you to reply.`
+			`	"The Hai have put out the call for volunteers to staff the new Mountaintop station when it's finished. While you've been off watching early construction works, we've been gauging public interest and well... it's not going to work. In order to sell this, they're going to need at least four thousand volunteers by the end of the week to get a good handle on what they can expect to need in terms of facilities, amenities, supplies, and various livability requirements. Nearly twelve thousand by the time the station is actually built. Our current estimates suggest they'll only get a quarter of that unless something changes."`
```

Comment:
Do these 4000 volunteers need to be there? If they've only got 1000 so far, and we need to go everywhere, I don't see how it's reasonably possible to achieve this. How long does it take to go straight from Hai-Home to Ultima Thule by hyperdrive?

Reply:
Changed it to month